presidio-analyzer/presidio_analyzer/analyzer_engine.py

```diff
@@ -59,10 +59,31 @@ def __init__(self, registry=None, nlp_engine=None,
         else:
             self.default_score_threshold = default_score_threshold
 
+    # pylint: disable=unused-argument
+    def GetAllRecognizers(self, request, context):
```

Comment:
how is this exposed to the grpc endpoint?

Reply:
talked. its not at this point. internal endpoint only;